src/plugins/expression/expression-tile.tsx

```diff
@@ -20,16 +22,28 @@ declare global {
 
 export const ExpressionToolComponent: React.FC<ITileProps> = observer((props) => {
   const content = props.model.content as ExpressionContentModelType;
-  const mathfieldRef = useRef<MathfieldElement>(null);
+  const mf = useRef<MathfieldElement>(null);
+  const trackedCursorPos = useRef<number>(0);
 
-  const handleChange = (e: any) => {
-    content.setLatexStr(e.target.value);
-  };
+  if (mf.current?.keybindings){
+    // Disable mathlive's undo with keyboard, as our undo picks up cmd+z
+    replaceKeyBinding(mf.current.keybindings, "cmd+z", "");
```

Comment:
Have you checked how mathlive handles Windows/ chromebook machines where it would be "control+z"? Perhaps if you print out the current keybinds we can see if it includes double entries for each item. Or perhaps it automatically translates from cmd to control when on windows.

Reply:
Thanks - I printed out the bound keystrokes for other platforms and then replaced them if not mac.